pkg/skaffold/deploy/helm.go

```diff
@@ -186,109 +203,152 @@ func (h *HelmDeployer) helm(ctx context.Context, out io.Writer, useSecrets bool,
 	cmd.Stdout = out
 	cmd.Stderr = out
 
+	logrus.Infof("calling: %v", cmd.Args)
 	return util.RunCmd(cmd)
 }
 
+// deployRelease deploys a single release
 func (h *HelmDeployer) deployRelease(ctx context.Context, out io.Writer, r latest.HelmRelease, builds []build.Artifact, valuesSet map[string]bool) ([]Artifact, error) {
-	releaseName, err := expandTemplate(r.Name)
+	releaseName, err := expand(r.Name, nil)
 	if err != nil {
 		return nil, errors.Wrap(err, "cannot parse the release name template")
 	}
 
-	isInstalled := true
-	if err := h.helm(ctx, ioutil.Discard, false, "get", releaseName); err != nil {
+	o := installOpts{
```

Comment:
nit: since this is used pretty far from its definition, call it something a bit more descriptive like `opts`